crates/holochain_p2p/Cargo.toml

```diff
@@ -35,9 +35,16 @@ tokio-stream = "0.1"
 holochain_util = { version = "^0.3.0-beta-dev.0", path = "../holochain_util" }
 
 [features]
+
+test_utils = [
+  "mock_network",
+  "kitsune_p2p/test_utils",
+  "ghost_actor/test_utils",
+]
+
 mock_network = [
+  "test_utils",
```

Comment:
this is now a circular feature dependency. it looks like it's not worth maintaining `mock_network` by itself.